src/video.cpp

```diff
@@ -14,7 +14,6 @@ extern "C" {
 #include <libavutil/mastering_display_metadata.h>
 #include <libavutil/opt.h>
 #include <libavutil/pixdesc.h>
-#include <libswscale/swscale.h>
```

Comment:
Is something else pulling this in for the swscale APIs below?

Reply:
Probably swscale from the video.h

Reply:
I can leave it in `video.cpp` as well, if that's preferred.